### DISCUSSION TITLE
How are the titles that show up in the screensaver chosen?

### DISCUSSION THREAD

**Discussion by SteveDinn** [jellyfin/jellyfin-androidtv#3711] [General]
I had always assumed that the titles that the screensaver showed were randomized from my entire library, but lately I've started to doubt that is the case. I have noticed that a specific selection of movies and shows tends to get repeated for a day or two before moving onto a new selection. Am I crazy for thinking this?

There also seem to be certain titles that show up more often than I think they should overall if the selection were truly random.

**Comment by nielsvanvelzen** [comment]
It's random, it does filter out items that have no backdrop or an age rating above PG13 (0.16) or what is configured in the preferences (0.17).

I believe there have been some reports that the servers "random" is not that random though.
